modules/ROOT/pages/monitoring/logging.adoc

```diff
@@ -1261,6 +1262,16 @@ See <<gql-error-information, GQL error information>> for details on the contents
 | The query plan.
 Included when xref:configuration/configuration-settings.adoc#config_db.logs.query.plan_description_enabled[`db.logs.query.plan_description_enabled`] is `true`.
 
+| executionPlanCacheKeyHash
+| The 32-bit hash of the cache key used to cache the execution plan.
+It is a fixed-length 8-character string of a hex-encoded integer value.
+Enabled by default only in the JSON format.
+If multiple query executions use the same cached execution plan, their hashes should match.
+For `event=start`, the value is 000000.
+
+| executableQueryCacheHit
+| Whether the query was executed using a cached execution plan.
```

Comment:
```suggestion
| Whether the query string matched a cached execution plan.
```

This is my attempt to a bit more precise, as this is specifically about a cache hit in the top-level outer query cache that matches directly from the query string to a cached execution plan. The query could still be executed using a cached execution plan even if this is false, e.g. if it hits an inner cache after parsing. Feel free to rephrase it.

Reply:
Maybe "query text" is better than "query string". I don't know what terminology is commonly used in the rest of the Cypher docs.